docs/hive-integration/import-installed-cluster.md

```diff
@@ -1,84 +1,169 @@
-# Import Installed Cluster
+# Import an existing OCP cluster as an Assisted Day 2 Cluster
 
-It is possible to import an existing installed OpenShift in order to be able to add more workers to it. (Day 2 operation)
+It is possible to take an existing OCP cluster and import this as a Day2 cluster into assisted so that additional workers may be added.
+Additional workers are added by downloading an ISO that has been generated from an InfraEnv linked to the imported cluster, the workers should be booted from this ISO.
 
-## CRDs
+This guide assumes that a Hub/Spoke configuration is being used and that the Hub has an installation of the Assisted Operator with Hive integration installed.
 
-Here are the needed CRDs to import an existing cluster:
+### Import the cluster and prepare it for the addition of Day2 workers:
 
-### InfraEnv
+#### 1: Create a namespace to which the imported cluster will belong.
 
-* [InfraEnv](crds/infraEnv.yaml) or
-* [InfraEnv Late Binding](crds/infraEnvLateBinding.yaml)
-
-### NMState Config
-
-Optional network configuration.
-* [NMState Config](crds/nmstate.yaml)
-
-### Pull Secret
-
-* [PullSecret Secret](crds/pullsecret.yaml)
-
-### Cluster Image Set
-
-* [ClusterImageSet](crds/clusterImageSet.yaml)
-
-### AgentClusterInstall
-
-Configure the number of control planes in the `provisionRequirements` field to be greater than `1`. This is needed because adding additional nodes to a Single Node OpenShift is not currently supported.
-
-* [AgentClusterInstall](crds/agentClusterInstall.yaml)
+```
+cat << EOF | oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: spoke-cluster
+EOF
+```
 
-### ClusterDeployment
+#### 2: Set up a cluster image set, this requires a release image and a name. It's important that this matches the cluster being imported.
```

Comment:
in ACM the infra owner is usually the one that create the cluster-image-set the one that creates the cluster may not have the ability to create image set.